src/Common/tests/XunitTraitsDiscoverers/Attributes/PlatformSpecificAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    /// <summary>
+    /// Apply this attribute to your test method to specify this is a platform specific test.
+    /// </summary>
+    [TraitDiscoverer("Xunit.TraitDiscoverers.PlatformSpecificDiscoverer", "XunitTraitsDiscoverers")]
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class PlatformSpecificAttribute : Attribute, ITraitAttribute
+    {
+        public PlatformSpecificAttribute(PlatformID platform) { }
```

Comment:
In some cases the platform-specificity will be temporary, linked to some issue.  How do we want to handle that?  Just add a comment next to the attribute on a test?  Or should we add a ctor that accepts an active issue, ala [ActiveIssue(...)]?  Or maybe a constructor should be added to ActiveIssue that includes a params array of PlatformIDs?